js/EnvironmentOverview.jsx

```diff
@@ -1,33 +1,47 @@
-var React = require("react");
-var Redux = require('redux');
-var ReactRedux = require('react-redux');
+const React = require('react');
+const Redux = require('redux');
+const ReactRedux = require('react-redux');
+const ReactRouter = require('react-router');
+const thunkMiddleware = require('redux-thunk').default;
+const createLogger = require('redux-logger');
 
-var thunkMiddleware = require('redux-thunk').default;
-var createLogger = require('redux-logger');
+const App = require('./containers/App.jsx');
+const DeployModal = require('./containers/DeployModal.jsx');
 
-var App = require('./containers/App.jsx');
+const reducers = require('./reducers/index.js');
+const webAPI = require('./_api.js');
+const actions = require('./_actions.js');
 
-var rootReducer = require("./reducers/index.js");
-var webAPI = require('./_api.js');
-var actions = require('./_actions.js');
+const createHistory = require('history').createHistory;
+const useRouterHistory = require('react-router').useRouterHistory;
 
-var store = Redux.createStore(
-	rootReducer,
+const store = Redux.createStore(
+	reducers,
 	Redux.applyMiddleware(
 		thunkMiddleware,
 		createLogger()
 	)
 );
 
-var Plan = function(props) {
+export function getDeploymentHook(store) {
```

Comment:
yeah, that's right. Any ideas on the name? It's basically taking the URL param and getting the deployment (or setting a new deployment).